spec/adapters/datatables_spec.rb

```diff
@@ -4,6 +4,8 @@
   before { Daru::View.table_library = :datatables }
   before(:each) do
     @data_array = [[1, 15], [2, 30], [4, 40]]
+    @string_array = [[1, 15], ["daru", "view"]]
```

Comment:
For googlecharts table we only use 1 data type  [Refer](https://github.com/SciRuby/daru-view/blob/master/lib/daru/view/adapters/googlecharts.rb#L163) . So we must raise error for googlecharts table.